[WIP] upgrade fastlane 2.154.0 to 2.164.0

jakubgs opened the pull request:
It's been a while and a fix for an issue I've reported a long time ago has been merged in https://github.com/fastlane/fastlane/pull/17456. With this fix we should see less iOS signing issues appearing as `errSecInternalComponent`.

This should also enable me to try a fix I proposed in https://github.com/fastlane/fastlane/issues/15185 which consists of creating a temporary unlocked keychain for each build.

State: __WIP__

jakubgs committed:
upgrade fastlane 2.154.0 to 2.164.0

It's been a while and a fix for an issue I've reported a long time ago
has been merged in https://github.com/fastlane/fastlane/pull/17456. With
this fix we should see less iOS signing issues appearing as `errSecInternalComponent`.

This should also enable me to try a fix I proposed in
https://github.com/fastlane/fastlane/issues/15185 which consists of
creating a temporary unlocked keychain for each build.

Signed-off-by: Jakub Sokołowski <jakub@status.im>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 4ccb70f5 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-11310/1/) | 2020-10-20 09:01:01 | ~15 min | `ios` | [:package:`ipa`](https://i.diawi.com/zdVCVZ) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FzdVCVZ)|

jakubgs committed:
nix: Increase stalled-download-timeout to 1 hour

Might help with issues described in: https://github.com/status-im/infra-ci/issues/17.

Details: https://github.com/NixOS/nixpkgs/issues/65015#issuecomment-662675716

Signed-off-by: Jakub Sokołowski <jakub@status.im>
